src/network/mcpe/handler/DeathPacketHandler.php

```diff
@@ -52,7 +55,8 @@ public function setUp() : void{
 		$parameters = [];
 		if($this->deathMessage instanceof Translatable){
 			$language = $this->player->getLanguage();
-			if(!$this->player->getServer()->isLanguageForced()){
+			$namespace = explode(".", $this->deathMessage->getText())[0];
+			if(!in_array($namespace, NamespacedLanguage::getNamespaces(), true) && !$this->player->getServer()->isLanguageForced()){
```

Comment:
This feels hacky, and I'm not a big fan of the in_array either. Seems to me that this should be achievable with a regular array<string, string>

Reply:
Please clarify what you're looking for here.